RosaP.txt

```diff
@@ -0,0 +1,5 @@
+alternative-reality new branch (mines called amend-rosa-perez
+
+Hello, my name is Rosy.  Three words that describe me are  mean, ugly, selfish.  

```

Comment:
I don't think those words describe you Rosy. You come off as a good person and you should think of yourself in far better light. It gets better.

Reply:
thank you.  This was the second pull request for the instructions for alternative reality, so describe opposite of myself.